packages/core/src/providers/MaestroProvider.ts

```diff
@@ -0,0 +1,194 @@
+import type { BitcoinNetwork } from "~/createConfig";
```

Comment:
Well we should probably cover this with tests to see if API responses match with the expected response. W/o it will quite hard to rely that the provider works.